### DISCUSSION TITLE
Why did the downloads increase so massively?

### DISCUSSION THREAD

**Discussion by BugInYourCode01** [BugInYourCode01/Create-Bamboo-Island#6] [Q&A] [answered]
I noticed that the daily downloads suddenly increased from around 5 per day to 200 per day (yesterday and today):
![image](https://github.com/user-attachments/assets/7c0c9f61-83f3-492c-aa16-7796cdd6c84f)
So I'm assuming that someone somewhere said or wrote something about this modpack that drew attention to it. The thing about it is that with every mention comes feedback, which helps me find and fix problems better. So if you came here from any other website, please write to me here. That would really help me a lot.

**Comment by kenshi-SQ** [comment, marked as answer]
I found this modpack in Tik Tok
https://www.tiktok.com/@dashnewtik/video/7450792261253598486?_r=1&u_code=dd8hfc889c3d1c&preview_pb=0&sharer_language=ru&_d=eegecia697gl6h&share_item_id=7450792261253598486&source=h5_m&timestamp=1734871930&user_id=6845717537050248198&sec_user_id=MS4wLjABAAAAVUr1UIF716zgndR51Cdv74PBGa-iaDsE69Ns2t0T_BDxf41za5HGYu6TmgzOsEqO&social_share_type=0&utm_source=copy&utm_campaign=client_share&utm_medium=android&share_iid=7450046976866158342&share_link_id=91b979d3-bc7f-41b3-b9c5-5003e0892ffb&share_app_id=1233&ugbiz_name=MAIN&ug_btm=b7360%2Cb2878&link_reflow_popup_iteration_sharer=%7B%22click_empty_to_play%22%3A1%2C%22dynamic_cover%22%3A1%2C%22follow_to_play_duration%22%3A-1.0%2C%22profile_clickable%22%3A1%7D&enable_checksum=1

**Reply by kenshi-SQ** [reply]
But this is Russian community)

**Reply by kenshi-SQ** [reply]
And this Russian community requires a Russifier

**Reply by BugInYourCode01** [reply]
Anyone can make a translation if they want, just edit the file https://github.com/MrRedstoneToGo/Create-Bamboo-Island/blob/main/datapacks/Create%20Sky%20Additions/assets/create_sky_additions/lang/en_us.json and send it to me. The only reason why I didn't take over the translation from Create: Bamboo Sky was because of the many changes (also in the translation file) which would have resulted in many translation errors.

**Reply by kenshi-SQ** [reply]
One guy in comment say he make translate, but the developer decided not to add

**Reply by kenshi-SQ** [reply]
I can make a translate, but I need you to contact me on discord and explain some blocks and how it works, otherwise I didn't figure it out myself

**Comment by kenshi-SQ** [comment]
Greetings again, I have made a translation into Russian language, it may be incorrect in places, but understandable
[ru_ru.json](https://github.com/user-attachments/files/18231975/ru_ru.json)

**Reply by BugInYourCode01** [reply]
Thank you very much for the translation. I will make sure that I publish a new version with the new changes (including translation) tomorrow.

**Comment by Vlada3333** [comment]
maybe it's not correct in some places, but it's understandable, thank you.

**Reply by BugInYourCode01** [reply]
Well, change it.

**Reply by kenshi-SQ** [reply]
I will change translate, but i need communication with you

**Reply by BugInYourCode01** [reply]
How do you imagine that? In minecraft or discord (I actually don't have it yet) or what else?
